nsiqcppstyle_util.py

```diff
@@ -66,6 +66,10 @@ def CmpObjects(a, b):
 
 def RemoveOuterQuotes(raw_string):
     final_string = raw_string.strip()
-    if final_string[0] == final_string[-1] and final_string[0] in [SINGLE_QUOTE, DOUBLE_QUOTE]:
+    if (
+        len(final_string) >= 2

```

Comment:
Can you give an example of the failure case where we need this check?

Reply:
As the commit comment states, this happen when the config file contains empty lines, that's one of the case I covered with the filefilter.example.txt